kostyor/db/migrations/alembic/env.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import with_statement
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+from logging.config import fileConfig
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+target_metadata = None
```

Comment:
Could you please elaborate on what exactly was wrong? You've got to have the same DDL anyway.

Besides, manual writing mean it's much easier to get out of sync database schema produced by ORM and database schema produced by Alembic. In that case having a special test to check that there's no difference is a must even on early stages.